deploy/examples/common.yaml

```diff
@@ -125,7 +125,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "create", "delete", "patch"]
+    verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
```

Comment:
why do we need PV update access? i don't see its required for external-provisioner/external-resizer

Reply:
CI was failing at deleting the PV.
```
E0112 09:10:40.783738       1 controller.go:1007] error syncing volume "pvc-d8b8c06c-49f2-4d71-b866-714164ee11fe": persistentvolumes "pvc-d8b8c06c-49f2-4d71-b866-714164ee11fe" is forbidden: User "system:serviceaccount:smoke-ns-system:rook-csi-rbd-provisioner-sa" cannot update resource "persistentvolumes" in API group "" at the cluster scope
```

we also have it in ceph-csi 
https://github.com/ceph/ceph-csi/blob/200fbdbf28591c404a85888d8bfc9561fa23dbb8/deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml#L25

Reply:
it might be dependent on the sidecar version. i have not seen this in latest version of sidecar here